crates/bitwarden/src/mobile/crypto.rs

```diff
@@ -172,7 +174,7 @@ pub async fn get_user_encryption_key(client: &mut Client) -> Result<String> {
         .get_key(&None)
         .ok_or(Error::VaultLocked)?;
 
-    Ok(user_key.to_base64())
+    Ok(user_key.to_base64().expose().to_owned())
```

Comment:
Should we add support to uniffi for Sensitive and avoid exposing them explicitly?

Reply:
Yeah that sounds reasonable, added it in a5a8097

Reply:
@Hinton I've had to add a uniffi Record using the new type to make sure it gets generated by uniffi correctly, which is a bit weird, have we hit this limitation before? 2c1ff1cc1e6f0f96bc626a2150dde73a2ed8010f